c10/xpu/XPUStream.cpp

```diff
@@ -232,14 +261,11 @@ XPUStream getStreamFromPool(const int priority, DeviceIndex device) {
     device = c10::xpu::current_device();
   }
   check_device_index(device);
-  TORCH_CHECK(
```

Comment:
Should we remove this check from all implementations (in particular the CUDA one)?
That way the python `torch.Stream(priority=1)`  would mean the lowest priority for both? And in particular using -10/10 like you do in your test would be a good cross-device strategy to get the highest/lowest priority?

Reply:
OK, I will remove CUDA's check in another PR.